BiasAndFairnessModule/Makefile

```
@@ -35,6 +35,11 @@ else
 	pylint $(shell find . -type f -name "*.py" ! -path "./.venv/*")
 endif
 
+test:
+	pytest tests --ignore=tests/base_test_metrics.py
+
+check: format lint
```

checkmake (0.2.2)

[warning] Target "check" should be declared PHONY.

(phonydeclared)